cpp/src/replace/replace.cu

```diff
@@ -81,16 +95,39 @@ namespace{ //anonymous
                     gdf_size_type      nrows,
```

Comment:
I would simplify this interface to just take `gdf_column*` arguments and access the data and valid pointers from the columns.

Reply:
simplified